### ISSUE TITLE
Auto-scroll to current chapter

### ISSUE THREAD

**Issue by NeeRaj-2401** [libre-tube/LibreTube#4566]
### Steps to reproduce

1. Open video (with chapters)
2. Click on chapter name which is in player controls

### Expected behavior

should Open chapters bottom sheet with scrolled and highlighted to clicked chapter name,
and 
if the chapter sheet is already open then scroll and highlight to chapter name which is clicked.

### Actual behavior

everytime we click on chapter name inside the player controls, a new instance of chapters bottom sheet gets created.

### LibreTube version

latest debug after #4563

### Android version

12

### Other details

The chapter highlighting part was there in previous chapters ui, hence i mentioned it.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by NeeRaj-2401** [comment]
There two more things to address releated to chapter's bottom sheet. Idk i should raise new issue or not hence mentioning here first

1. Minimising player with chapter's sheet open should actually minimise/close the chapters sheet ( just like comments)
2. Chapters card's inner layout have some boxy look, the lighter part should be bit rounded 

<img src="https://github.com/libre-tube/LibreTube/assets/88180887/9611d58d-1180-488f-9ef9-c88bc4691b97" width="40%">

**Comment by Bnyro** [comment]
That screenshot looks really strange, that's not how it's supposed to look and how it does look on my phone. Weird Android stuff again.

**Comment by Bnyro** [comment]
should be fixed now :)

**Comment by NeeRaj-2401** [comment]
Yup the chapter card part looks fine now :)
But
1. clicking on player's chapter name open/close chapter sheet, doesn't auto-scroll to current chapter name

2. While interacting with player with chapter sheet open, the minimising player behaviour is still not same as it is with comments sheet

**Comment by Bnyro** [comment]
1. I've already been looking into that auto-scrolling thing to the current chapter, but there seems to be an Android bug that it just doesn't scroll  at all.
2. Is the intended behavior. Imo, chapters have a different relevance than comments and hence should be dismissed first before minimizing the player.

**Comment by NeeRaj-2401** [comment]
1. Aah okay, thanks :)

2. But imo, keeping the same behaviour brings **consistency** cause personally i got confused for a sec that why player didn't get minimised . 
( also if user wants to minimise tha chapter sheet , there is already bar for it just below the Player )
